solutions/security/ai/attack-discovery.md

```diff
@@ -22,9 +22,11 @@ For a demo, refer to the following video (click to view).
 
 ## Role-based access control (RBAC) for Attack Discovery [attack-discovery-rbac]
 
-You need the `Attack Discovery: All` privilege to use Attack Discovery.
+You need the `Attack Discovery: All` privileges to use Attack Discovery.
 
-![attack-discovery-rbac](/solutions/images/security-attck-disc-rbac.png)
+{applies_to}`serverless: ` You need the `Rules: Read` privilege at a minimum to use Attack Discovery.
```

Comment:
@yctercero in Serverless, will users need the `Attack Discovery: All` privilege _and_ the `Rules: Read` privilege to use Attack Discover? Or do they only need the `Rules: Read` privilege? The updated screenshot (`security-attck-disc-rbac.png`) makes me think both, but I don't want to assume.

Reply:
Correct, we require both. [here](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/attack_discovery/links.ts#L20) is an example, where we only show Attack Discovery links in the sidebar when you have both `Rules: Read` and `AttackDiscovery`